examples/extensions/commands/clean/cmd_clean.py

```diff
@@ -0,0 +1,68 @@
+from conan.api.conan_api import ConanAPIV2
+from conans.cli.command import conan_command, OnceArgument
+from conans.cli.output import Color, ConanOutput
+from conans.client.userio import UserInput
+
+info_color = Color.BRIGHT_WHITE
+recipe_color = Color.BRIGHT_BLUE
+removed_color = Color.BRIGHT_YELLOW
+
+
+@conan_command(group="Custom commands")
+def clean(conan_api: ConanAPIV2, parser, *args):
+    """
+    Deletes (from local cache or remotes) all recipe and package revisions but
+    the latest package revision from the latest recipe revision.
+    """
+    parser.add_argument('-r', '--remote', action=OnceArgument,
+                        help='Will remove from the specified remote')
+    parser.add_argument('--force', default=False, action='store_true',
+                        help='Remove without requesting a confirmation')
+    args = parser.parse_args(*args)
+
+    def confirmation(message):
+        """
+        Returns True if "--force" was passed or if user answers "yes" to the question message, otherwise, False
+        """
+        return args.force or ui.request_boolean(message)
+
+    def remove_all_recipe_revisions_but_latest_one(recipe_, rrevs):
```

Comment:
I think there's no need to use these functions, also we want to give visibility to the api functions, I would just simplify the code, remove output and keep it to the minimum that a user can follow easily...